src/meta-types/intersection/object.ts

```diff
@@ -121,12 +128,17 @@ type IntersectObjects<
     ObjectOpenProps<META_OBJECT_A>,
     ObjectOpenProps<META_OBJECT_B>
   >,
+  INTERSECTED_CLOSE_ON_RESOLVE = And<
```

Comment:
Intersection: is this the expected behaviour for the `closeOnResolve` option?

Reply:
It made more sense to use a `Or` instead :)